pyodide-build/pyodide_build/buildall.py

```diff
@@ -719,6 +723,15 @@ def copy_packages_to_dist_dir(
             output_dir / dist_artifact_path.name, compression_level=compression_level
         )
 
+        if metadata_files and dist_artifact_path.suffix == ".whl":
```

Comment:
Could you move this (extracting metadata file from a wheel) into a separate function, and add some tests for the function? I think you can put it in `common.py`.

Reply:
Thank you for the suggestions - I've extracted a helper function and added two tests for it.